msibi/tutorials/README

```diff
@@ -5,4 +5,9 @@ code works as expected.
 LJ : recover known LJ potential
     - This test, as described in the original paper, is recovering a
       known potential to show that the methodology works. The target
-      data was derived from LJ fluids at three different states. 
+      data was derived from LJ fluids at three different states.
+
+PROPANE : recover known propane potential
+    - This test, as described in the original 2014 paper, is recovering a
+      known potential to show that the methodology works. The target
```

Comment:
Close. We actually are doing coarse-graining here (3-site propane to single-site propane), so this is a bit misleading. You can use section III.C of [this paper](http://aip.scitation.org/doi/abs/10.1063/1.4880555?journalCode=jcp) for reference.